packages/core/helper-plugin/src/hooks/useRBAC.ts

```diff
@@ -106,14 +106,14 @@ export const useRBAC = (permissionsToCheck, passedPermissions) => {
    * until all the checks were complete.
    */
   const allowedActions = (
-    data.some((res) => res === undefined) ? defaultAllowedActions : data
+    data.some((res) => res === undefined) ? defaultAllowedActions : (data as DefaultAllowedActions)
```

Comment:
You shouldn't have to cast this, instead you can pass a generic to the `post` call to define the response :)